imod/tests/fixtures/mf6_circle_fixture.py

```diff
@@ -340,3 +340,85 @@ def circle_model_transport():
     simtimes = pd.date_range(start="2000-01-01", end="2001-01-01", freq="W")
     simulation.create_time_discretization(additional_times=simtimes)
     return simulation
+
+
+
+@pytest.fixture(scope="function")
+def circle_model_transport_multispecies():
+
+    al = 0.001
+    porosity = 0.3
+    max_concentration = 35.0
+    min_concentration = 0.0
+    max_density = 1025.0
+    min_density = 1000.0
+    species = ["sali", "tempe"]
+
+    simulation = make_circle_model_flow_with_transport_data(species)
+    gwf_model = simulation["GWF_1"]
+
+
+    for specie in species:
+
+        transport_model = imod.mf6.GroundwaterTransportModel()
+        transport_model["ssm"] = imod.mf6.SourceSinkMixing.from_flow_model(
+            gwf_model, specie, save_flows=True
+        )
+        transport_model["disv"] = gwf_model["disv"]
+
+        # %%
+        # Now we define some transport packages for simulating the physical processes
+        # of advection, mechanical dispersion, and molecular diffusion dispersion. This
+        # example is transient, and the volume available for storage is the porosity,
+        # in this case 0.10.
+
+
+        transport_model["dsp"] = imod.mf6.Dispersion(
+            diffusion_coefficient=1e-4,
+            longitudinal_horizontal=al,
+            transversal_horizontal1=al * 0.1,
+            transversal_vertical=al * 0.01,
+            xt3d_off=False,
+            xt3d_rhs=False,
+        )
+        transport_model["adv"] = imod.mf6.AdvectionUpstream()
+        transport_model["mst"] = imod.mf6.MobileStorageTransfer(porosity, save_flows=True)
+
+        # %%
+        # Now we define initial conditions (0.0) and output options for the transport
+        # simulation, and assign the transport model to the simulation as well.
+        transport_model["ic"] = imod.mf6.InitialConditions(start=max_concentration)
+        transport_model["oc"] = imod.mf6.OutputControl(
+            save_concentration="all", save_budget="all"
+        )
+
+        simulation[f"tpt_{specie}"] = transport_model
+    slope = (max_density - min_density) / (max_concentration - min_concentration)    
+    modelnames = [f"tpt_{specie}" for specie in species]    
+    gwf_model["buoyancy"] = imod.mf6.Buoyancy(
+        reference_density=min_density,
+        modelname=modelnames,
+        reference_concentration=[min_concentration,min_concentration ],
+        density_concentration_slope=[slope/2, slope/2],
```

Comment:
Why the division by 2?

Reply:
the original model had 1 species, now we have 2 that have the same initial conditions and boundaries. So if I were to use the same slope, the water would get 2 times heavier. 
But indeed that is not an issue perse, so now I  removed the "/2"